common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -130,6 +136,13 @@ def is_proctored_exam(self):
         """ Alias the is_proctored_enabled field to the more legible is_proctored_exam """
         return self.is_proctored_enabled
 
+    @property
+    def allow_unproctored_exam(self):
```

Comment:
Reminder: Please update this function name and the comment below once @catong helps settle language questions, because it is confusing as is.